glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt

```diff
@@ -7,6 +7,32 @@ package mozilla.telemetry.glean.config
 import mozilla.components.concept.fetch.Client
 import mozilla.components.lib.fetch.httpurlconnection.HttpURLConnectionClient
 import mozilla.telemetry.glean.BuildConfig
+import mozilla.telemetry.glean.rust.toByte
+
+import com.sun.jna.Structure
+import com.sun.jna.ptr.LongByReference
+
+@Structure.FieldOrder("dataDir", "packageName", "uploadEnabled", "maxEvents")
+class FfiConfiguration(
```

Comment:
nit: can this be marked as `internal`? What do you think of creating an `internal` method in `Configuration` that returns a pre-configured structure out of a `Configuraiton` object?

Reply:
Or even just an extension function for `Configuration` (still internal)

Reply:
Yes, I think it should be possible to have a conversion method on Configuration that creates the FFI configuration

Reply:
re Configuration object: We can't. The configuration object passed in by users into Glean.initialize actually doesn't hold all data we need to pass on (such as the package name and the data path)